.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,33 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    export REGION_TAG_PARSER_DIR="/tmp/region-tag-parser"
+    export PARSER_PATH="${REGION_TAG_PARSER_DIR}/wizard-py/cli.py"
+    export PIP_PATH=/tmp/pyyaml
+    export PYTHONPATH="$PYTHONPATH:$PIP_PATH"
+
+    XUNIT_PATH="$PWD/sponge_log.xml"
+    XUNIT_TMP_PATH="/tmp/drift_tmp.xml"
+
+    if [[ -f "$XUNIT_PATH" ]]; then
+        echo "=== Injecting region tags into XUnit output ==="
+        pip install pyyaml -q -t $PIP_PATH
```

Comment:
Why do you need `$PIP_PATH`?

Reply:
The script wasn't able to find this library without it being installed in a specific directory, that is then added to the `$PYTHONPATH`.

(There might be cleaner workarounds, but this was the easiest one to implement. And given that these are largely-ephemeral CI machines, I didn't think "leaving some stuff behind" like this would be all that bad.)

Reply:
Hm. Can we run `pip install --user pyyaml` in `run_tests.sh`?

If we install it in `run_single_test.sh`, it will be installed over 200 times.

Reply:
Done.